src/content/docs/logs/ui-data/query-syntax-logs.mdx

```diff
@@ -58,11 +72,11 @@ General query rules:
       </td>
 
       <td>
-        When a term contains special characters, double-quote the term and escape the special characters using a backslash (`\`). This includes special characters such as `+`, `-`, `&`, `|`, `!`, `(`, `)`, `{`, `}`, `[`, `]`, `^`, `"`, `~`, `*`, `?`, `:`, `/`, or `\`.
+        When a term contains special characters, double-quote the term and escape the special characters using a backslash (`\`). This includes special such as `+`, `-`, `&`, `|`, `!`, `(`, `)`, `{`, `}`, `[`, `]`, `^`, `"`, `~`, `*`, `?`, `:`, `/`, or `\`.
 
-        Example: To query for `"POST /log/v1 HTTP/1.1" 202`, escape the quotes like this:
+        Example: To query for a `status` attribute containing exactly `"POST /log/v1 HTTP/1.1" 202`, escape the quotes like this:
 
-        `"\"POST /log/v1 HTTP/1.1\" 202"`
+        `status: "\"POST /log/v1 HTTP/1.1\" 202"`
```

Comment:
I'm turning all these into attribute queries, since I think the "plain term means message contains" thing is confusing when you first run into it 🤔

Reply:
thanks for making a more useful query!